transport_nantes/photo/static/photo/js/vote_system.js

```diff
@@ -0,0 +1,120 @@
+function getCookie(name) {
```

Comment:
This would be instance four in our code base of exactly the same function?

Reply:
That's a leftover apparently, I moved into the previous commit this function to our base template